rasa_core/tracker_store.py

```diff
@@ -250,3 +256,137 @@ def retrieve(self, sender_id):
 
     def keys(self):
         return [c["sender_id"] for c in self.conversations.find()]
+
+
+class SQLTrackerStore(TrackerStore):
+    """Store which can save and retrieve trackers from an SQL database"""
+
+    from sqlalchemy.ext.declarative import declarative_base
+    Base = declarative_base()
+
+    class SQLEvent(Base):
+        __tablename__ = 'conversations'
```

Comment:
should this rather be `events`?

Reply:
Yeah I went back and forth about this...although `events` is probably better since when this is in production, the db will have multiple conversations stored insequentially